etstool.py

```diff
@@ -134,6 +139,8 @@
     envvar="ETSTOOL_EDM",
 )
 
+github_url_fmt = "git+http://github.com/enthought/{0}.git#egg={0}"
```

Comment:
I would make `source_dependencies` a dict and specify the complete pip/github url as the value.  That is simpler and more general.

Reply:
addressed.

Reply:
note that i made the same change in traitsui as well